Update docs for new external ID retrieval workflow (#938)
astronomer/docs: astro/create-cluster.md

> If not specified, Astronomer will create a cluster with two `m5.xlarge` nodes and a maximum node count of 20 in `us-east-1` by default. For information on all supported regions, configurations, and defaults, see [AWS resource reference](resource-reference-aws.md).

Reviewer: ```suggestion
If you don’t provide the necessary information, Astronomer support creates a cluster with two `m5.xlarge` nodes and a maximum node count of 20 in `us-east-1`. For information about supported AWS regions, configurations, and defaults, see [AWS resource reference](resource-reference-aws.md).
```

Author: This sounds a bit accusatory, the information isn't "necessary" because a customer could be fine to just create a cluster with the default settings in any given category

Reviewer: @jwitz I'm not sure how _necessary_ is accusatory! Perhaps _If you don't provide all of the information_ would work better. You can't start the sentence with _If not specified_ as the subject isn't defined.  Maybe _If you don't provide some or all of the configuration information_ ?

> If not specified, Astronomer will create a cluster with two `m5.xlarge` nodes and a maximum node count of 20 in `us-east-1` by default. For information on all supported regions, configurations, and defaults, see [AWS resource reference](resource-reference-aws.md).

Reviewer 2: ```suggestion
If you do not specify configuration preferences, Astronomer will create a cluster with two `m5.xlarge` nodes and a maximum node count of 20 in `us-east-1`. For information on all supported regions, configurations, and defaults, see [AWS resource reference](resource-reference-aws.md).
```

From there, your Astronomer representative will provide you with a unique `External ID` for each new cluster. Make note of this value for the next step.

#### Step 2: Edit your AWS trust policy

In the AWS IAM console, [edit the `astronomer-remote-management` trust relationship](https://docs.aws.amazon.com/directoryservice/latest/admin-guide/edit_trust.html) to include new trust policies and ensure that Astronomer has permission to manage your new cluster.

> To do this, add the External ID that corresponds to your new cluster to the existing `sts:ExternalId` grouping. For example, your policy for two Astro clusters might look like the following, with `<External-ID-2>` being the External ID for your new cluster:

Author: Just curious, does the change mean that Astronomer can modify this policy directly? Why do we no longer need this step?

          "Condition": {
            "StringEquals": {

>               "sts:ExternalId": [

Reviewer 2: ```suggestion
```

Does this get removed too?

>                 ...

Reviewer 2: ```suggestion
                "<External-ID>"
```

    aws iam set-security-token-service-preferences --global-endpoint-token-version v2Token
    ```

#### Step 2: Confirm with Astronomer

> A notification is sent from Astronomer when your cluster is created. After your cluster is created, you can create a new Deployment in the cluster. See [Create a Deployment](create-deployment.md).

Reviewer: ```suggestion
Astronomer support sends you a notification when your cluster is created. After your cluster is created, you can create a new Deployment in the cluster. See [Create a Deployment](create-deployment.md).
```

    aws iam set-security-token-service-preferences --global-endpoint-token-version v2Token
    ```

#### Step 2: Confirm with Astronomer

> Astronomer support sends you a notification when your cluster is created. After your cluster is created, you can create a new Deployment in the cluster. See [Create a Deployment](create-deployment.md).

Reviewer 2: ```suggestion
Astronomer support sends you a notification when your cluster is created. After your cluster is created, you can create a new Deployment in the cluster and start deploying pipelines. See [Create a Deployment](create-deployment.md).
```
